eslint_test.json

```diff
@@ -1,4 +1,5 @@
 {
+    "parser": "babel-eslint",
```

Comment:
Suggestion (optional): It would be nice to document why this parser is needed in case it can be removed, however this is a JSON configuration file which doesn't support comments. I'm not sure if it's feasible to convert it to a `.js` configuration file, so I'm marking this suggestion as optional.

Reply:
Ah, I believe this comment is outdated by your next PR which converts it to `.yml`.

Reply:
This dependency gets updated to `@babel/eslint-parser` by #568. That PR also ports the ESLint configs to YAML in order to allow them to be later commented.